test/smoke/smoke-tests.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const testConfig = require('./config');
+const testURL = `${testConfig.url}/details/404`;
+
+describe('Smoke Tests', () => {
+
+  before(() => {
+    require('../acceptance/mixins/login')(browser);
```

Comment:
This needs to be inside here because the required `browser` object doesn't exist until the tests are ready to start. Otherwise, yeh I'd put it at the top!